tests/data/preview/remove_for_brackets.py

```diff
@@ -1,3 +1,6 @@
+for (((x))) in points:
```

Comment:
Can you add some tests where the assignment target is too long to fit in one line? Like `for (many, variables, that, go, on, over, eighty, eight, characters) in some_list:`.

Reply:
Done, 51e5be8dafcc97ec5c6f15ec51bcb8aa02b48acb.